api/base/settings/defaults.py

```diff
@@ -189,3 +189,6 @@
 ENABLE_ESI = osf_settings.ENABLE_ESI
 VARNISH_SERVERS = osf_settings.VARNISH_SERVERS
 ESI_MEDIA_TYPES = osf_settings.ESI_MEDIA_TYPES
+
+ADDONS_FOLDER_CONFIGURABLE = ['box', 'dropbox', 's3', 'googledrive']
```

Comment:
Is hard-coding this consistent with our configuration mechanism for addons in general?